lib/routes/admin-api/feature.js

```diff
@@ -9,6 +9,18 @@ const {
 } = require('../../permissions');
 
 const version = 1;
+const fields = [
+    'name',
+    'description',
+    'type',
+    'project',
+    'enabled',
+    'stale',
+    'strategies',
+    'variants',
+    'created_at',
+    'last_seen_at',
+];
```

Comment:
I feel this is leaking the column names from the stores to the controller. Could we just define the fields as they are returned from the stores/serivices and have the store perform the column mapping themselves (from camelCase to snake_case).

Reply:
good point; the controller shouldn't actually know the column names. I'm open to suggestions as to how to do this.

Reply:
ca6e07a82b69f65bb4fbfa2d45e3101acef3da8a seems like a good solution.